frontend/packages/dev-console/src/utils/catalog-utils.ts

```diff
@@ -0,0 +1,70 @@
+import {
```

Comment:
We should not add a dependency from other plugins to `dev-console`.

Please move this utils and their tests into `packages/console-shared/src/components/catalog/utils/catalog-utils.tsx`